.dumi/theme/common/ChangeModal.tsx

```
@@ -0,0 +1,255 @@
+      }),
+    [],
+  );
+
+  // 检测溢出状态
+  React.useEffect(() => {
+    const container = containerRef.current;
+    if (!container) return;
+
+    checkOverflow();
+
+    const resizeObserver = new ResizeObserver(checkOverflow);
```

ESLint

[error] ResizeObserver is not supported in op_mini all, KaiOS 2.5

(compat/compat)